cypress/integration/patron-notice-policies/create-patron-notice.spec.js

```diff
@@ -0,0 +1,23 @@
+import { TextField, Button, TextArea } from '../../../interactors';
+
+describe('FAT-1455 test', () => {
+  beforeEach('login', () => {
+    cy.login(Cypress.env('diku_login'), Cypress.env('diku_password'));
+    cy.visit('/settings');
+  });
+  afterEach(() => {
+    cy.get('.stripes__icon.icon-triangle-down.medium---jNVX7').click();
+    cy.do(Button({ id: 'dropdown-clickable-delete-item' }).click());
+    cy.do(Button({ id: 'clickable-delete-item-confirmation-confirm' }).click());
+  });
+  it('create new patron notice', () => {
+    const nameExample = 'Test notice';
+    cy.get('.label---zHSX1').contains('Circulation').click();
```

Comment:
Please, try to re-implement using interactors. Plus same as above - wrap business actions and move into fragments.